src/components/ComponentPages/CampForum/UI/ThreadListUI.tsx

```diff
@@ -116,7 +116,10 @@ const ThreadListUI = ({
           key="title"
           render={(text, others) => {
             return (
-              <a onClick={(e) => onThreadClick(e, others)}>
+              <a
```

Comment:
Probably we can use Link

Reply:
here we using onclick event if we use Link then we need to define href but here href is not available